### PR TITLE
Add File-Managers to Useful Utilities

### PR DESCRIPTION
**Pull request by 299792458ms**
- add File-Managers.md with weight 10 to Useful Utilities
- update Other.md weight from 10 to 11
- insert link to File-Managers in the _index.md
- fix punctuation of first commit

### CODE REVIEW

File: pages/Useful Utilities/File-Managers.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+weight: 10
+title: File Managers
+---
+
+## GUI
+
+- [dolphin](https://github.com/KDE/dolphin): File manager by KDE.
+- [nautilus](https://apps.gnome.org/en/Nautilus/): A file browser for Gnome.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:9]: "by KDE" but "for Gnome"

pick one pls

File: pages/Useful Utilities/File-Managers.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - [nautilus-admin-gtk4](https://github.com/MacTavishAO/nautilus-admin-gtk4): Open files with elevated priviledges.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:10]: privileges

File: pages/Useful Utilities/File-Managers.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    - [nautilus-image-converter](https://gitlab.gnome.org/coreyberla/nautilus-image-converter): Resize and rotate images in Nautilus.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:11]: "in nautilus" is redundant..

File: pages/Useful Utilities/File-Managers.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    - [nautilus-open-any-terminal](https://github.com/Stunkymonkey/nautilus-open-any-terminal): Open terminals in selected directory.
+- [nemo](https://github.com/linuxmint/nemo): File browser for Cinnamon.
+    - [nemo-fileroller](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-fileroller): File archiver extension for Nemo.
+    - [nemo-terminal](https://github.com/linuxmint/nemo-extensions/tree/master/nemo-terminal): Embedded terminal window for Nemo.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:15]: again "for nemo" is redundant

File: pages/Useful Utilities/File-Managers.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- [pcmannfm](https://github.com/lxde/pcmanfm): Extremely fast and lightweight file manager.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:16]: misspelled name

File: pages/Useful Utilities/File-Managers.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    - [nerd-fonts](https://github.com/ryanoasis/nerd-fonts): For icon support.
+    - [ffmpegthubmnailer](https://github.com/dirkvdb/ffmpegthumbnailer): For video thumbnails,
+    - [p7zip(7-zip)](https://github.com/p7zip-project/p7zip): For archive extraction and preview.
+    - [jq](https://jqlang.github.io/jq/): For JSON preview.
+    - [poppler](https://gitlab.freedesktop.org/poppler/poppler): For PDF preview.
+    - [fd](https://github.com/sharkdp/fd): For file searching.
+    - [ripgrep](https://github.com/BurntSushi/ripgrep): For file content searching.
+    - [fzf](https://github.com/junegunn/fzf): For quick subtree navigation.
+    - [zoxide](https://github.com/ajeetdsouza/zoxide): For historical directories navigation.
+    - [wl-clipboard](https://github.com/bugaevc/wl-clipboard): For system clipboard support.
```
Comment (vaxerski) [inline pages/Useful Utilities/File-Managers.md:34]: TUI stuff have tons of extensions, no point in having all of them here, I'd drop all of these, considering they are more like dependencies.

File: pages/Useful Utilities/File-Managers.md (hunk 1 above)
Comment (299792458ms) [reply pages/Useful Utilities/File-Managers.md:9]: They are the official description in their respective project repo. Change them anyway for consistency?

File: pages/Useful Utilities/File-Managers.md (hunk 6 above)
Comment (299792458ms) [reply pages/Useful Utilities/File-Managers.md]: done

**Comment by 299792458ms** [comment]
did all the recommended changes

**Review by fufexan — APPROVED** [review]

File: pages/Useful Utilities/File-Managers.md (hunk 1 above)
Comment (vaxerski) [reply pages/Useful Utilities/File-Managers.md]: yea

**Review by vaxerski — APPROVED** [review]

**Pull request merged**